py3status/modules/taskwarrior.py

```diff
@@ -5,6 +5,8 @@
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 5)
     format: display format for this module (default '{task}')
+    task_args: arguments passed to the command
```

Comment:
(Opinion) Maybe we should use `filter` following the `task` manual.

Reply:
I like `args` as this can be more than a filter (in my case I use it to override some configuration options), but `filter` is good too

Reply:
I don't use `task`. It looks like a `rc:` filter (to me, at least) to override configuration options. I'm wondering if we want `filter` or `command` because you can do this too... `task export +ACTIVE`.

With your `if self.filter is not None`, we will get this...
```
3d [chris:~] 2 % task '' export                                                                                                        
No expression to evaluate.
```
```
3d [chris:~] % task export | wc -l                                                                                                     
13    # 13 tasks
```